src/serializer.cpp

```diff
@@ -88,6 +88,21 @@ std::string realastic_impl::serialize_open_event(struct_open_syscall *event)
     return ss.str();
 }
 
+std::string realastic_impl::serialize_creat_event(struct_creat_syscall *event)
+{
+    std::string buff;
+    if (event->rc > 0)
```

Comment:
please check the length of pathname and in order to prevent messy console just report maximum 32 character(kMaximumOutputBufferSize) :
if (strlen(event->pathname) < kMaximumOutputBufferSize)
    {
        buff = std::move(escape_special_charachter(std::string(event->pathname, event->pathname+strlen(event->pathname))));
        buff += "\"";
    }
    else
    {
        buff = std::move(escape_special_charachter(std::string(event->pathname, event->pathname+ kMaximumOutputBufferSize)));
        buff += "\"...";
    }